composer.json

```diff
@@ -11,12 +11,13 @@
     ],
     "require": {
         "php": ">=5.3.9",
-        "puli/repository": "1.0.*",
+        "puli/repository": "^1.0",
         "puli/discovery": "^1.0",
         "puli/url-generator": "^1.0",
         "puli/manager": "^1.0",
         "webmozart/console": "^1.0",
         "webmozart/path-util": "^2.0",
+        "webmozart/key-value-store": "^1.0.0-beta5",
```

Comment:
I'm not sure about this. I think the cli needs the key-value store for path-mapping repositories but it was included as discovery also needed it. However, we set the dependency in discovery as a suggestion, so I think we have to put that here for things to work properly. @webmozart ?